doc/source/computational_model.rst

```diff
@@ -111,14 +111,14 @@ The individuals' flow utility is given by:
 
 	U(l_t, \theta, Z_t) =
 	\begin{cases}
-	0, & \text{if $l_t = O$,}
+	0, & \text{if $l_t = N$,}
 	\\[4pt]
 	\theta_l + Z'_t\alpha(l_t), & \text{if $l_t = P$ or $F$},
 	\end{cases}
 
 where :math:`\alpha(l_t) = \alpha_F + \alpha_P \cdot \bf{1}` :math:`(l_t = P)`.
 
-The CRRA utility depends on consumption per adult equivalent, female labor supply :math:`l`, characteristics :math:`Z`, and {\diw preference for work :math:`\theta`}. :math:`Z` can contain information on marital status, presence of children, their interaction, dummies for children in different age groups, an indicator wether or not the partner is working, etc. :math:`U(.)` of not working is normalised to zero; :math:`\beta` is set to 0.98;
+The CRRA utility depends on consumption per adult equivalent, female labor supply :math:`l`, characteristics :math:`Z`, and {\diw preference for work :math:`\theta`}. :math:`Z` can contain information on marital status, presence of children, their interaction, dummies for children in different age groups, an indicator weather or not the partner is working, etc. :math:`U(.)` of not working is normalised to zero; :math:`\beta` is set to 0.98;
```

Comment:
Please fix spelling. https://dictionary.cambridge.org/dictionary/english/whether-or-not

Reply:
Done